docs/customization.md

```diff
@@ -0,0 +1,79 @@
+# <img alt="OpenSK logo" src="img/OpenSK.svg" width="200px">
+
+## Customization
+
+### 3D printed enclosure
```

Comment:
See previous comment: the enclosure only applies to the Nordic dongle so I would put it there instead.

Reply:
Moved